enabler/test/de/schildbach/pte/live/FrenchSouthWestProviderLiveTest.java

```diff
@@ -0,0 +1,193 @@
+/*
+ * Copyright 2014-2015 the original author or authors.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package de.schildbach.pte.live;
+
+import static org.junit.Assert.assertTrue;
+
+import org.junit.Test;
+
+import de.schildbach.pte.FrenchSouthWestProvider;
+import de.schildbach.pte.dto.Point;
+
+/**
+ * @author Nicolas Derive
+ */
+public class FrenchSouthWestProviderLiveTest extends AbstractNavitiaProviderLiveTest
+{
+	public FrenchSouthWestProviderLiveTest()
+	{
+		super(new FrenchSouthWestProvider(Secrets.NAVITIA_AUTHORIZATION));
+	}
+
+	@Test
+	public void nearbyStationsAddress() throws Exception
+	{
+		nearbyStationsAddress(44826434, -0557312);
```

Comment:
Literals starting with 0 are interpreted as octal. Are you sure this is what you want to do?